broot/builder.py

```diff
@@ -49,6 +49,7 @@ def _setup_yum(self, mirror):
                             line = line.replace("https", "http")
 
                     name_to_key = {"fedora-20": "246110C1",
+                                   "fedora-23": "246110C1",
```

Comment:
I think that that whole peice of code is a bad idea.  Downloading keys over plain http!

On a normal fedora system, it is like:

    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch

That is provided by the fedora-repos package.  The issue is probably that you need to have the gpg key there before you verify any packages, including fedora-repos.  Maybe we need to copy the keys across?  This needs testing and investigation.